tests/schema/tests.py

```diff
@@ -318,6 +318,35 @@ def test_fk_db_constraint(self):
             editor.alter_field(Author, new_field2, new_field, strict=True)
         self.assertForeignKeyNotExists(Author, 'tag_id', 'schema_tag')
 
+    @isolate_apps('schema')
+    @skipUnlessDBFeature('supports_foreign_keys', 'can_introspect_foreign_keys')
+    def test_no_db_constraint_added_during_field_change(self):
+        """
+        When a primary key that's pointed to by a ForeignKey with
+        db_constraint=False is altered, a foreign key constraint isn't added.
+        """
+        class Author(Model):
+            class Meta:
+                app_label = 'schema'
```

Comment:
This is because `tests` will be used for `app_label` if not explicitly specified.